qiita_pet/templates/study_description_templates/prep_template_panel.html

```diff
@@ -9,24 +9,31 @@
         <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
         <h4 class="modal-title" id="myModalLabel">Choose preprocessing parameters</h4>
       </div>
-      <form action="/study/preprocess" method="post" onsubmit='return submit_preprocess();'>
+      <form action="/study/preprocess" method="post" onsubmit='return submit_preprocess({{prep_id}});'>
         <div class="modal-body">
           <div>
             <div class="form-group">
               <table>
-                {% for idx, form_item in enumerate(preprocess_form, start=1) %}
-                  <tr>
-                    <td width="50%">{% raw form_item.label %}</td>
-                    <td width="50%">{% raw form_item(tabindex=idx, class_='form-control') %}</td>
-                  </tr>
-                {% end %}
                 <tr>
                   <td>
+                    <select id="preprocessing_parameters_{{prep_id}}" name="preprocessing_parameters_id" class="form-control" onchange="$('#display-extra-info-{{prep_id}}').html($(this).children(':selected').attr('extra-info'))">
+                      <option value="0" extra-info="">Select an option ...</option>
+                      {% for idx, name, text in preprocess_options %}
```

Comment:
We are guaranteed to always have preprocessing options right? so this would never be a dropdown menu that's essentially empty?